File: Android/PantallasApp/app/src/main/java/com/example/pantallasapp/MainActivity.java
package com.example.pantallasapp;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    Button botonAceptar;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        botonAceptar = (Button) findViewById(R.id.boton_aceptar);

        botonAceptar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent otraPantalla;
                otraPantalla = new Intent(getApplicationContext(), OtraActivity.class);
                startActivity(otraPantalla);
            }
        });

    }
}

File: Android/personaSQLite/app/src/main/java/com/example/personotasqlite/MainActivity.kt
package com.example.personotasqlite

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val btnIngresa: Button = findViewById(R.id.ingresa)
        val btnVer: Button = findViewById(R.id.ver)

        btnIngresa.setOnClickListener() {
            val ingresaPapu : Intent = Intent(applicationContext, ingresaPersona::class.java)
            startActivity(ingresaPapu)
        }

        btnVer.setOnClickListener() {
            val verPapu : Intent = Intent(applicationContext, vePersona::class.java)
            startActivity(verPapu)
        }
    }
}